spec/controllers/concerns/two_factor_authenticatable_methods_spec.rb

```diff
@@ -117,7 +117,7 @@
             end
 
             it 'creates a new user event with disavowal' do
-              expect(UserAlerts::AlertUserAboutNewDevice).to receive(:send_alert) do |**args|
+              expect(UserAlerts::AlertUserAboutNewDevice).to receive(:send_alert) do |**_args|
```

Comment:
we could prob just drop the args altogether here?
```suggestion
              expect(UserAlerts::AlertUserAboutNewDevice).to receive(:send_alert) do
```